locales-src/newtab.ftl

```diff
@@ -83,6 +83,13 @@ newtab-menu-delete-history = Delete from History
 newtab-menu-save-to-pocket = Save to { -pocket-brand-name }
 newtab-menu-delete-pocket = Delete from { -pocket-brand-name }
 newtab-menu-archive-pocket = Archive in { -pocket-brand-name }
+newtab-menu-show-privacy-info = Our sponsors & your privacy
+
+## newtab privacy info modal
+newtab-privacy-modal-done = Done
```

Comment:
How is this used? Is it a button, a link? It would be good to have the role in the message ID.

Reply:
Can you give me an example of what you mean?

Reply:
Ah, ok, thanks!